.env

```diff
@@ -46,3 +46,4 @@ REACT_APP_USER_SERVICE_CURRENT=/api/v1/users/current/
 REACT_APP_USER_SERVICE_LOGOUT=/api/v1/users/logout/
 
 REACT_APP_STATUS_SERVICE=/api/v1/status/
+REACT_APP_AUTH_TOKEN_SERVICE=/api/v1/token/
```

Comment:
required:
if this is replacing an actively used path you'll also need to update the
`.env.development` file with the same `param` but using the original path for development

Reply:
this is required. V1 UI didn't need it because login was done through the django template view for login.

updated `.env.development` accordingly, thanks for pointing this out =D